alsa_settings/README.md

```diff
@@ -0,0 +1,73 @@
+# ALSA settings
+
+SOF test case execution requires certain DSP and mixer configurations being set
+on the DUT (Device Under Test). An important part of this configuration is the
+signal levels adjusted to the hardware setup which runs the tests, including its
+codec and loopback devices.
+
+The `./alsa_settings/` directory contains custom ALSA configurations which
+are currently applied at the SOF CI pipeline's hardware platforms.
```

Comment:
Why is the _default_ hardware configuration different and not enough? If this is by design (in other words, testing requires some "special sauce"), very briefly state why. If this is not design, clearly state that all these files are really "workarounds".

Either way, it's not good to be testing something different from regular users. So this deserves a bit of justification in either case.